src/ui/common/src/components/pages/workflows/index.tsx

```diff
@@ -87,19 +87,30 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
         });
       }
 
+      let containsWarning = false;
       let checks = [];
       if (workflow.checks) {
         checks = workflow.checks.map((check) => {
           const value =
             check.result?.exec_state.status === 'succeeded' ? 'True' : 'False';
+          const level = check.spec?.check?.level ?? CheckLevel.Warning;
```

Comment:
Is there a world in which this wouldn't be set?

Reply:
I'm not entirely sure 🤔 Just moved around what was there. 

@likawind Could this happen when there is a check that hasn't run yet?
Or maybe a situation where the spec isn't set?

Reply:
We shouldn't reach the fall-back in practice. I think this is just a safeguard where the type is completely missing, and in this case something is seriously wrong in backend.

I've seen multiple complaints about this line being confusing, alternatively we can just remove the check level icon if the level is undefined